src/platforms/apple/common/enriching-events/user-feedback.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "User Feedback"
+sidebar_order: 105
+
+description: "Learn more about the user feedback API, which gives you the possibility to collect user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+---
+
+The user feedback API gives you the possibility to collect user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues.
+Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. To get the `eventId` you for example [`beforeSend`](/platforms/apple/guides/ios/configuration/options/#before-send) or use the return value of the methods capturing an event.
```

Comment:
```suggestion
Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. To get the `eventId` for example, you can use the [`beforeSend`](/platforms/apple/guides/ios/configuration/options/#before-send) or the return value of the methods capturing an event.
```

Reply:
Whopsi, I was already gaga yesterday in the evening 🙈